server/player/player.go

```diff
@@ -54,6 +54,9 @@ type playerData struct {
 	armour                       *inventory.Armour
 	heldSlot                     *uint32
 
+	rideable  entity.Rideable
```

Comment:
I'd call this "riding" instead of "rideable", it currently doesn't sound like it's the entity the player is riding

Reply:
Still not clear imo, I think you need to have "riding" in there somewhere. If you want you can also add a doc to explain but i don't think that's necessary with just "riding" or "ridingEntity". You could also do "riddenEntity" or something like that. The "able" just sounds like it's something that can be ridden, not actively being ridden